### DISCUSSION TITLE
Install/Uninstall elastic agent

### DISCUSSION THREAD

**Discussion by camdg12** [Security-Onion-Solutions/securityonion#12538] [Unsupported Versions]
Hello,
I use a distributed architecture: a manager, a fleet node, a search node and a forward node, and I want to install elastic agents in Windows endpoints, but I get the following error: "no Fleet Hosts are accessible- check connectivity to Fleet Host"( ports are open between the endpoint & the fleet: 8220,5505)
Also, Could you please tell us how to uninstall elastic agent from windows endpoint. The error is: failed to stop service (Elastic Agent)

**Comment by TotieBash** [comment]
Watch the youtube link below and you would be an expert.  Check the elastic agent install log and make sure you have name resolution to the `https://<SO>:8220` whatever host name that is on the log file.

https://youtu.be/cGmQMsFuAvw?si=ZtwqyOn5a9gFHupr
